src/rust/engine/process_execution/src/remote.rs

```diff
@@ -33,6 +33,8 @@ use workunit_store::WorkunitStore;
 // module.
 mod streaming;
 pub use streaming::StreamingCommandRunner;
+#[cfg(test)]
+mod streaming_tests;
```

Comment:
Will the existing tests be able to be ported to streaming? There is a lot of good coverage there...

Reply:
I wanted to land the bug fix earlier than porting the remainder, but yes I intend to port the bulk of the unit tests.